### PR TITLE
Add network interface binding documentation

### PR DESCRIPTION
**Pull request by mohsenrezaeithe**
Fixes https://github.com/favonia/cloudflare-ddns/issues/1006

This would help users of the Docker container approach to be able to bind their updater to a specific network interface and get the right public IP.

### CODE REVIEW

**Comment by favonia** [comment]
Thank you. I have two broad suggestions:

1. I prefer not to clutter the beginning template, especially when the setting `network_mode:` might conflict with `networks:`.
2. Perhaps we should warn that this may bypass quota limitation on a particular network interface. That is, the specially routed traffic might not count towards to the quota set in the host.

**Comment by mohsenrezaeithe** [comment]
> 1. I prefer not to clutter the initial template, especially when the setting `network_mode:` might conflict with `networks:`.

Updated.

> 2. Perhaps we should warn that this might bypass quotas or firewalls for a specific network interface. For instance, the specially routed traffic might not count towards the quota set on the host.

I don't follow. I'm not familiar with any OS level quotas that'd affect this.

**Comment by favonia** [comment]
> > 3\. Perhaps we should warn that this might bypass quotas or firewalls for a specific network interface. For instance, the specially routed traffic might not count towards the quota set on the host.
> 
> I don't follow. I'm not familiar with any OS level quotas that'd affect this.

I wasn't being clear—I was talking about `iptables` and `nftables`. You can set up quotas or other rules on the host, and I believe `macvlan` will bypass them. In other words, the host (almost) can't see the traffic.

**Comment by mohsenrezaeithe** [comment]
> I wasn't being clear—I was talking about `iptables` and `nftables`. You can set up quotas or other rules on the host, and I believe `macvlan` will bypass them. In other words, the host (almost) can't see the traffic.

Makes sense. I've added a note in the docs.

File: README.markdown [hunk 1]
```diff
@@ -179,6 +186,35 @@ The easiest way to enable IPv6 is to use `network_mode: host` so that the update
 
 </details>
 
+<details>
+<summary><em>Click to expand:</em> 🛜 Bind to a specific network interface for updates</summary>
+
+📜 This method uses a MacVLAN sub-device to bind to a specific network interface and may interfere with your custom iptables and nftables configurations.
```
Comment (favonia) [inline README.markdown:192]:
```suggestion
📜 This method uses a MacVLAN sub-device to bind to a specific network interface and may bypass your custom `iptables` and `nftables` configurations.
```

**Comment by favonia** [comment]
@mohsenrezaeithe Thank you for your contribution.

**Pull request merged**